src/BenchmarkDotNet/EventHandlers/IEventHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Results;
+using BenchmarkDotNet.Validators;
+
+namespace BenchmarkDotNet.EventHandlers
+{
+    public interface IEventHandler
```

Comment:
`EventHandler` has a very specific meaning in C#: https://learn.microsoft.com/en-us/dotnet/api/system.eventhandler?view=net-7.0

Adding an `I` in front of it may suggest that it's an interface that `EventHandler` implements.

I believe that we should find a better name. I don't have any good suggestions, but perhaps @bartonjs could throw us a hand here?

Reply:
Event...Processor?  Event...Broker?

Reply:
EventListener?

Reply:
what about `BenchmarkEventHandler`? I'm going to use this for now but if we want to go with something else then am happy to change it again